src/state/apis/zapper/client.ts

```diff
@@ -33,7 +33,7 @@ export const CHAIN_ID_TO_ZAPPER_NETWORK_MAP = invert(ZAPPER_NETWORKS_TO_CHAIN_ID
   Record<ChainId, SupportedZapperNetwork>
 >
 
-export const zapperNetworkToChainId = (network: SupportedZapperNetwork): ChainId | undefined =>
+export const zapperNetworkToChainId = (network: SupportedZapperNetwork): ChainId =>
```

Comment:
this type sig change is wrong - this can absolutely return undefined, especially as we have to cast to key into this map

Reply:
cdc6d37b9016ba65b7cb12e6717e1a4c633cad23